### PR TITLE
Add RFC: platform-practice-4ps.md

### PR DESCRIPTION
**Pull request by luizonthedrums**
This is the RFC for the platform practice, after discussions in the platform practice and with @erikdstock @mc-jones and @joeyAghion

### CODE REVIEW

File: RFCs/platform-practice-4ps.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    
+
+## People:
+
+* It's open to everybody in PDDE and part of the engineering team calendar: Encourage inclusivity by making the practice accessible to all team members.
+
+### Roles of this practice:
+
+#### 1. Facilitator:
+   * The Facilitator plays a crucial role in managing the process of the platform practice. This includes time-keeping, organizing the agenda, reviewing takeaways and action items, and diligently documenting sessions in Notion. The Facilitator ensures the smooth flow and effectiveness of each session.
```
Comment (joeyAghion) [inline RFCs/platform-practice-4ps.md:41]: > diligently documenting

I might loosen this to documenting decisions or follow-ups. It's possible we've been _too_ diligent, noting everything discussed at the expense of engaging fully in the discussion.

File: RFCs/platform-practice-4ps.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+*   Presentation or discussion of the topics (time-boxed 10-15 min):
+
+    
+    * Ensure focused discussions within a specified time frame.
+    * Verify if there's a need to continue on the topic: Assess whether additional discussion or action is required for a particular topic.
+    * Another time-box (8-10min): Further discussions or actions within a constrained timeframe.
+    * Wrap-up of the topic: Summarize key points and decisions made during the discussion.
+    
+
+-   Conclusion of the meeting: Recap important takeaways and action items, and formally conclude the practice session (it's ok to not have any action item due to the context of the topics)
```
Comment (joeyAghion) [inline RFCs/platform-practice-4ps.md:65]: Maybe clarify who leads each of these? With 3 "roles" there's a risk of diffuse responsibility.

**Review by joeyAghion — APPROVED** [review]

File: RFCs/platform-practice-4ps.md (hunk 1 above)
Comment (luizonthedrums) [reply RFCs/platform-practice-4ps.md]: Agreed, I changed the description.

File: RFCs/platform-practice-4ps.md (hunk 2 above)
Comment (luizonthedrums) [reply RFCs/platform-practice-4ps.md]: Updated

File: RFCs/platform-practice-4ps.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+   * The Facilitator plays a crucial role in managing the process of the platform practice. This includes time-keeping, organizing the agenda, documenting decisions, and following up on action items. The Facilitator ensures the flow and effectiveness of each session.
+
+#### 2. Champion:
+   * The Champion is the driving force behind topics discussed in the platform practice. They bring forth specific subjects with clear objectives, whether seeking a decision or fostering an open discussion. The Champion actively guides discussions, aiding the Facilitator in achieving meaningful conclusions for each topic.
+
+#### 3. Curator:
+   * The Curator takes on the responsibility of shaping the overall direction of the platform practice. They provide valuable advice and curate relevant topics that align with the practice's goals. The Curator's role is crucial in maintaining the practice's focus and ensuring that discussions contribute to the overall improvement and coherence of the platform.
+
+
+## Process:
```
Comment (MounirDhahri) [inline RFCs/platform-practice-4ps.md:50]: Something I appreciated recently was reminding engineers that the meeting is happening in X minutes with a clear agenda
Screenshot 2024-02-13 at 11 20 57

**Review by mc-jones — APPROVED** [review]
Very nice doc.

**Comment by luizonthedrums** [comment]
https://www.notion.so/artsy/Platform-Practice-1b558d0627444c9c9bf7ed7583767ca2 page udpated

**Pull request merged**